homeassistant/components/vesync/fan.py

```diff
@@ -140,17 +140,27 @@ def extra_state_attributes(self) -> dict[str, Any]:
         if hasattr(self.device.state, "active_time"):
             attr["active_time"] = self.device.state.active_time
 
-        if hasattr(self.device.state, "display_status"):
+        if (
+            hasattr(self.device.state, "display_status")
+            and self.device.state.display_status is not None
```

Comment:
Is it possible that the `display_status` is changing from a value to `None`, or will it only be `None` when the model does not support it? The question is it could happen that the value will change at runtime to `None` or not

Attributes should not be dynamically added/removed from the attributes, as automating on them will be a lot more complex.

Please answer this question for the others as well.